.storybook/main.js

```diff
@@ -18,6 +18,17 @@ module.exports = {
 			name: "storybook-addon-turbo-build",
 			options: {
 				optimizationLevel: 1,
+				// We're explicitly setting the minification options below because
+				// we want to turn off `minifyIdentifiers`. Why? Because it breaks
+				// Islands hydration. When you minify the component filenames
+				// the dynamic imports fail to find them.
+				esbuildMinifyOptions: {
+					target: "es2015",
+					minify: false,
+					minifyWhitespace: true,
+					minifyIdentifiers: false,
+					minifySyntax: true,
```

Comment:
Why can't we just send over 
```TS
{ minifyIdentifiers: false }
```

as the comment above suggests?

Reply:
The impression I get from the docs on this is that once you turn off minify you need to be explicit about these sub options

https://github.com/privatenumber/esbuild-loader#minify

I added some links to the comment to make this clearer